tests/e2e/pageObjects/CreateSpacePage.js

```diff
@@ -0,0 +1,53 @@
+require("dotenv").config();
+
+const { format } = require("util");
+
+const config = require("../../cucumber.conf.js");
+
+class CreateSpacePage {
+
+  constructor() {
+    this.baseUrl = config.tandoorURL;
+    this.searchURL = this.baseUrl + `search/`;
+    this.spaceOverviewURL = this.baseUrl + `space-overview`;
```

Comment:
use join. it can help with missing forward slash
```suggestion
    this.searchURL = path.join(this.baseUrl, 'search');
    this.spaceOverviewURL = path.join(this.baseUrl, 'space-overview');
```